doc/source/changed_in_57.rst

```diff
@@ -0,0 +1,263 @@
+=======================================================================================
+
+Some features that were present in |Percona Server| 5.6 have been removed in |Percona Server| 5.7. These are:
+
+ * `Handlersocket <https://www.percona.com/doc/percona-server/5.6/performance/handlersocket.html>`_ - This feature might be included in a future release if HandlerSocket starts supporting 5.7.
+ * `Support for Fake Changes <https://www.percona.com/doc/percona-server/5.6/management/innodb_fake_changes.html>`_ - Instead of slave prefetching using the fake changes, a 5.7 intra-schema parallel replication slave should be used.
+ * ``SHOW ENGINE INNODB STATUS`` no longer prints the count of active Read-Only transactions.
+ * `InnoDB redo log archiving <https://www.percona.com/doc/percona-server/5.6/management/log_archiving.html>`_ has been removed due to lack of user uptake of the feature.
+
+Changed features that were present in Percona Server 5.6
+=========================================================
+
+Some features that were present in |Percona Server| 5.6 have been replaced by a different implementation of the same/similar functionality in |Percona Server| 5.7. These are:
+
+ * The minor |Percona Server| version number ("y" in "5.a.b-x.y") has been dropped to simplify |Percona Server| versioning.
+
+ * Performance Schema memory instrumentation support has been added to the :ref:`audit_log_plugin`, :ref:`scalability_metrics_plugin`, and :ref:`pam_plugin`, and to track memory used by :ref:`user_stats`, :ref:`per_query_variable_statement`, :ref:`changed_page_tracking`, and :ref:`threadpool` features.
+
+ * :ref:`audit_log_plugin` now produces diagnostics in a format consistent with the rest of the server, having been converted to use |MySQL| plugin services API.
+
+ * The `performance_schema.metadata_locks <https://dev.mysql.com/doc/refman/5.7/en/metadata-locks-table.html>`_ table now displays ``backup`` and ``binlog`` lock information too. The ``object_type`` column has two new valid values: ``backup``, and ``binlog``.
+
+ * :table:`XTRADB_RSEG` table schema has been changed to support new possible |InnoDB| page sizes. The ``zip_size`` column has been removed and replaced by new columns physical_page_size, logical_page_size, and is_compressed.
+
+ * :table:`XTRADB_READ_VIEW` table no longer contains the ``READ_VIEW_UNDO_NUMBER`` column, which was associated with unused code and always contained zero.
+
+ * Interaction between ``--hidden-`` option modifier and `session_track_system_variables <http://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_session_track_system_variables>`_: any variables with ``--hidden-`` modifier become hidden from the latter variable too. Thus, they should not be present there. Even if you never set ``session_track_system_variables``, care must be taken if a variable contained in its default value (i.e. autocommit) is hidden.
```

Comment:
I am not sure my blueprint note ("Interaction between X and Y: blah blah") was the best possible phrasing stylistically